app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -138,6 +144,184 @@ fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
     }
 }
 
+@Composable
+fun FrontCardFace(id: String) {
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(CategoryLevelDummy.valueOf(level).backgroundColor),
+        contentAlignment = Alignment.Center,
+    ) {
+        val cardImage = painterResource(id = R.drawable.bg_card_large)
+        val category = painterResource(id = R.drawable.ic_tag_love)
+
+        Image(
+            painter = cardImage, contentDescription = null, modifier = Modifier.fillMaxSize()
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)